CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+### 2.1 
+ * **IMPORTANT UPGRADE NOTE FOR CRUNCHY PACKAGE USERS**: In version 2.0, the Crunchy provided extras for node_exporter were split out from the pgmonitor-pg##-extras package. A dependency was kept between these packages to make upgrading easier. For 2.1, the dependency between these packages has been removed. When upgrading from 1.7 or earlier, if you have node_exporter and postgres_exporter running on the same systems, ensure that you install the separate pgmonitor-node_exporters_extras package after the update. See the README for the full package name(s).
+
+ * Minium required versions of software used in pgmonitor have been updated to: Prometheus 2.5.0, Prometheus Alertmanager 0.15.3, postgres_exporter 0.4.7 (enables full PG11 support) & Grafana 5.3.4.
+ * Fixed Grafana data source to use the "proxy" mode instead of "direct" with default install. Should fix issues encountered with the default setup when Grafana is not installed on the same system as Prometheus.
```

Comment:
I think issue is not this one "when Grafana is not installed on the same system as Prometheus."

Issue with direct mode is that browser client will try to access prometheus over network and may fail if prometheus port is not opened which may be case when both are installed on same system as well when grafana and prometheus are behind corporate firewall and browser client outside.

Reply:
Ok. Reworded to:
```
 * Fixed Grafana data source to use the "proxy" mode instead of "direct" with default install. Should fix connection issues encountered during default setup between Grafana & Prometheus.
```